pkgs/development/interpreters/racket/default.nix

```diff
@@ -60,6 +60,8 @@ stdenv.mkDerivation rec {
     for p in $(ls $out/bin/) ; do
       wrapProgram $out/bin/$p --set LD_LIBRARY_PATH "${LD_LIBRARY_PATH}";
     done
+    export PATH="$out/bin:$PATH";
```

Comment:
@henrytill No, it's just so that raco on the next line has the $PATH it needs. Probably can be changed to `PATH=… raco` on the same line?